manual/CHAPTER_Overview.tex

```diff
@@ -193,6 +193,12 @@ \subsection{RTLIL Identifiers}
 
 All RTLIL identifiers are case sensitive.
 
+Some transformations, such as flattening, may have to change identifiers provided by the user
+to avoid name collisions. When that happens, attribute ``{\tt hdlname}`` is attached to the object
+with the changed identifier. The contents of this attribute follows the same rules as RTLIL
+identifiers, and in addition, multiple levels of hierarchy are represented by separating multiple
```

Comment:
nitpick - can you add the 'single space character' convention to the `hdlname` bullet point in the README too?

Reply:
Done, thanks for pointing out!